package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "eventsource",
+  "name": "@harnessio/event",
   "version": "2.0.2",
```

Comment:
This will be a `2.1.0` release since it adds a new feature

Reply:
Cheers done